lib/Alchemy/Phrasea/WorkerManager/Command/WorkerRunServiceCommand.php

```diff
@@ -39,14 +39,14 @@ protected function doExecute(InputInterface $input, OutputInterface $output)
             if ($body === false) {
                 $output->writeln(sprintf('<error>Unable to read payload file %s</error>', $input->getArgument('body')));
 
-                return;
+                return 0;
             }
 
             $body = json_decode($body, true);
             if (json_last_error() !== JSON_ERROR_NONE) {
                 $output->writeln('<error>Invalid message body</error>');
 
-                return;
+                return 0;
```

Comment:
why not returning a error code ? (1 ?) see: [https://hpc-discourse.usc.edu/t/exit-codes-and-their-meanings/414](https://hpc-discourse.usc.edu/t/exit-codes-and-their-meanings/414)

same thing todo for every failure point (catch's)